cmudb/extensions/tscout/tscout_feature_gen.py

```diff
@@ -0,0 +1,160 @@
+import re
+import sys
+from pathlib import Path
+from typing import List, Tuple
+
+from clang.cindex import TypeKind
+
+# We're assuming that this script is housed in postgres/cmudb/extensions/tscout
+# We calculate the path of TScout relative to this extension and add it to the PythonPath temporarily.
+TSCOUT_EXTENSION_PATH = Path(__file__).parent
+CODEGEN_TEMPLATE_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, "operating_unit_codegen.c")
+CODEGEN_FILE_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, "operating_unit_features.h")
+
+TSCOUT_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, Path("../../tscout/"))
+sys.path.append(TSCOUT_PATH.resolve().__str__())
+
+# And now, we can import TScout
+from tscout import model
+
+OU_TO_FEATURE_LIST_MAP = {}
+OU_EXCEPTIONS = {
+    "ExecHashJoinImpl": "T_HashJoin"
+}
+OU_EXCLUDED_FEATURES = [
+    "query_id",
+    "left_child_plan_node_id",
+    "right_child_plan_node_id",
+    "statement_timestamp"
```

Comment:
```suggestion
    "statement_timestamp",
```

Just future proofing.